src/sig_stfl/lms/external/hash.c

```diff
@@ -95,7 +98,8 @@ void hss_finalize_hash_context(int h, union hash_context *ctx, void *buffer) {
     if (hss_verbose) {
         printf( " -->" );
         int i; for (i=0; i<32; i++) printf( " %02x", ((unsigned char*)buffer)[i] );
-        printf( "\n" );
+        printf( "
```

Comment:
Did something go wrong with the `\n` symbols here?

Reply:
Yep, will fix.